tests/queries/0_stateless/02000_join_on_const.sql

```diff
@@ -43,6 +43,8 @@ SELECT * FROM t1 RIGHT JOIN t2 ON NULL ORDER BY t1.id NULLS FIRST, t2.id SETTING
 SELECT '- full -';
 SELECT * FROM t1 FULL JOIN t2 ON NULL ORDER BY t1.id NULLS FIRST, t2.id SETTINGS join_use_nulls = 1;
 
+SET query_plan_use_new_logical_join_step = 0;
```

Comment:
Why it has to be explicitly disabled?

Reply:
I assumed that I had enabled these queries for the new step since they are now planned as if the user wrote an equality with two constant columns from the left and right tables. However, I didn’t enable them for other algorithms anyway. For now, I’ve just removed `SET query_plan_use_new_logical_join_step = 0;`